plugins/mcreator-localization/help/default/potioneffect/category.md

```diff
@@ -0,0 +1,7 @@
+This parameter determines the effect category

```

Comment:
We usually format like this:

![image](https://github.com/MCreator/MCreator/assets/16374228/e0a7b11a-7e93-4bda-9eff-69181d9e34a6)